app/controllers/albums.js

```diff
@@ -1,18 +1,21 @@
 const logger = require('../logger');
 const axios = require('axios');
 const { Purchase } = require('../models');
-const { albumAlreadyPurchased, defaultError } = require('../errors');
+const { albumAlreadyPurchased, defaultError, dbError } = require('../errors');
 
 module.exports = {
   list(req, res, next) {
-    axios.get('https://jsonplaceholder.typicode.com/albums').then(function(response) {
-      res.status(200).json({
-        albums: response.data.map(album => {
-          delete album.userId;
-          return album;
-        })
-      });
-    });
+    axios
+      .get('https://jsonplaceholder.typicode.com/albums')
```

Comment:
as I told you previously maybe we could move all the external api logic inside a service and abstract it there.
Also we should move the base url of the external api to a configurable value, so you can easily change between different environments like development, stage, qa or production without changing the code.

Reply:
done in #14